tests/weather-fetcher.spec.js

```diff
@@ -0,0 +1,13 @@
+const should = require('should');
+const weatherFetcher = require('../models/weather-fetcher.js');
+
+describe("Weather fetcher", function() {
+  it('promise return value should exist', function() {
+    return weatherFetcher.fetch().then(function(data){
+        var items = data["rss"]["channel"]["item"]["description"]["body"]["data"];
```

Comment:
I'm not sure what is data and items. Can you give more meaningful names to these variables?